package.json

```diff
@@ -41,59 +42,72 @@
     ]
   },
   "dependencies": {
-    "@sambego/storybook-state": "^1.3.2",
+    "@babel/core": "^7.8.7",
+    "@babel/node": "^7.8.7",
+    "@babel/preset-env": "^7.8.7",
+    "@babel/preset-react": "^7.8.3",
+    "@babel/preset-typescript": "^7.8.3",
+    "@jest/core": "^24.0.0",
     "bootstrap": "~4.3.0",
     "classnames": "^2.2.6",
     "date-fns": "^1.30.1",
-    "react": "16.9.0",
-    "react-dom": "16.9.0",
+    "react": "^16.8.3",
```

Comment:
Why did this change? Can we keep it on 16.9.0 to match the main platform?

Reply:
I guess it's part of my test when I had mismatch with enzyme and I forgot to put it back